### PR TITLE
Add new mojom interfaces PathService and UpdateService.

### PR DESCRIPTION
**Pull request by mpcomplete**
These will be used by the updater package.

### CODE REVIEW

File: sky/services/updater/update_service.mojom [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module updater;
+
+interface UpdateService {
```
Comment (abarth) [inline sky/services/updater/update_service.mojom]: Don't you need a BUILD.gn change to actually get this to compile?

**Comment by abarth** [comment]
LGTM

File: sky/services/updater/update_service.mojom (hunk 1 above)
Comment (mpcomplete) [inline sky/services/updater/update_service.mojom]: Oops, pruned too many files. Added back.

Do I also need to check in the changes to sky/shell/BUILD.gn and sky/packages/sky_services/BUILD.gn that depend on this interface? Or is that part of step 2?

Comment (abarth) [inline sky/services/updater/update_service.mojom]: I'd add them in this CL.  It's only the Dart code that needs to wait.

Comment (mpcomplete) [inline sky/services/updater/update_service.mojom]: Done.

**Pull request merged**